### ISSUE TITLE
Secondary Screen Issue

### ISSUE THREAD

**Issue by AramShabanian** [DeskTranslate/DeskTranslate#15]
Greetings!

First off, FANTASTIC TOOL. This is a great help to me. I use a website called vidgrid (http://vidgrid.tk.gg) to stream up to 9 news feeds simultaneously, and your tool is phenomenally useful for translating the news tickers at the bottom of videos in different languages. That said, I cannot seem to get the tool to operate on any other than my primary monitor. Further, the ticker moves so fast that the text is sometimes hard to parse, and would be nice to have in a notepad document for posterity. 

Is there any (easy) way to accomplish this? Don't want to add a ton of requests to an already great tool, just figured others might find the same use I do and might appreciate this if it's simple! 

Thank you again! :)

**Comment by Drubaloo** [comment]
Were you able to find a fix? It still only operates on the primary screen
